test/sys/test_mman.rs

```diff
@@ -120,3 +120,36 @@ fn test_mremap_shrink() {
     // The first KB should still be accessible and have the old data in it.
     assert_eq!(slice[ONE_K - 1], 0xFF);
 }
+
+#[test]
+#[cfg(target_os = "netbsd")]
+fn test_mmap_prot_mprotect() {
+    use nix::errno::Errno;
+    use nix::libc::size_t;
+    use nix::sys::mman::mprotect;
+    use std::num::NonZeroUsize;
+
+    const ONE_K: size_t = 1024;
+    let ten_one_k = NonZeroUsize::new(10 * ONE_K).unwrap();
+    let prot = ProtFlags::PROT_READ;
+    prot.PROT_MPROTECT(ProtFlags::PROT_READ);
+
+    unsafe {
+        let ptr = mmap::<BorrowedFd>(
```

Comment:
Anonymous mapping has been remvoed from the `mmap(2)` interface, please use `mmap_anonymous()` instead